spec/chitter_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "./lib/chitter"
+
+describe Chitter do
+
+  describe "#all" do
+    it "Returns a chitter object, which we can extract a message instance from" do
+      @test = Chitter.new(:message => "Test 1", :posted_by => "Florence", :created_at => Time.now)
+      @test.save
+      array = Chitter.all
+      chitter_object = array[@test.id - 1]
+      expect(chitter_object.message).to eq("Test 1")
```

Comment:
So, I totally understand the urge to write these tests - however, they're not really necessary! You can adequately test the database component of your app with feature tests, writing unit tests like this you're really just testing DataMapper itself, which isn't your code.